roboteam_ai/src/utilities/GameStateManager.cpp

```diff
@@ -13,7 +13,21 @@ roboteam_msgs::RefereeData GameStateManager::getRefereeData() {
 
 void GameStateManager::setRefereeData(roboteam_msgs::RefereeData refMsg) {
     GameStateManager::refMsg = refMsg;
-    strategymanager.setCurrentRefGameState(static_cast<RefCommand>(refMsg.command.command));
+    auto cmd = static_cast<RefCommand>(refMsg.command.command);
+    //if we have shootouts, we need different handling
+    if (static_cast<int>(refMsg.stage.stage) == roboteam_msgs::RefereeStage::PENALTY_SHOOTOUT) {
+        if (cmd == RefCommand::PREPARE_PENALTY_US) {
```

Comment:
why do you need this only for penalty shootouts?

Reply:
I am confused, what are you referring to? ShootOuts have different rules than normal penalties.

Reply:
So even though we get the same refCommand we have to interpret it differently based on the ref stage.